### ISSUE TITLE
[bug] Lines 210 to 222 may cause an install abort on Ubuntu, git bash, & WSL when using auto installer

### ISSUE THREAD

**Issue by codesport** [rainbow-me/rainbowkit#651]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### RainbowKit Version

0.4.6

### wagmi Version

version installed with pnpm create @rainbow-me/rainbowkit@latest

### Current Behavior

Lines 210 to 222 cause an install abort on Ubuntu & WSL when using:

```
npm init @rainbow-me/rainbowkit@latest
# or
yarn create @rainbow-me/rainbowkit@latest
# or
pnpm create @rainbow-me/rainbowkit@latest
```
Therefore, removed highly opinionated git init and other actions on lines 210 to 222.

At least for git on Linux, WSL, and git bash on windows, you'll need to commit to master before attempting to rename branch to main

Otherwise it will cause an error because there is no reference to HEAD to any branch if nothing as been committed yet.

And, you can try also replicate this by manually creating repo running on Ubuntu or WSL. Try to rename the default branch without committing and you'll get a HEAD not found ref error.

Rainbow give this error:

```
  shortMessage: 'Command failed with exit code 128: git branch -m main',
  command: 'git branch -m main',
  escapedCommand: 'git branch -m main',
  exitCode: 128,
  signal: undefined,
  signalDescription: undefined,
  stdout: '',
  ```
Which is because of the head issue and renaming a branch with 0 commits

### Expected Behavior

It may or may not be good idea to automatically perform repo commits on end user's local system. However, Next.js does it and it works great!

### Steps To Reproduce

And, you can try also replicate this by manually creating repo running on Ubuntu, git bash for windows, or WSL. Try to rename without committing and you'll get a HEAD not found ref error.

`mkdir test && cd test && git init && git branch -m hotdog`

### Link to Minimal Reproducible Example (CodeSandbox, StackBlitz, etc.)

On WSL DO:  `mkdir test && cd test && git init && git branch -m main`

### Anything else?

Pull request to remove those lines:
https://github.com/rainbow-me/rainbowkit/pull/650

**Comment by markdalgleish** [comment]
This issue should be fixed in [create-rainbowkit v0.1.2](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Fcreate-rainbowkit%400.1.2) as we no longer forcibly rename the branch.

**Comment by codesport** [comment]
> This issue should be fixed in [create-rainbowkit v0.1.2](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Fcreate-rainbowkit%400.1.2) as we no longer forcibly rename the branch.

Thanks @markdalgleish .  Will reinstall and test it out. Appreciate the follow-up!

**Comment by codesport** [comment]
@markdalgleish confirming the issues was fixed.
